patches/server/0005-Paper-config-files.patch

```diff
@@ -1574,6 +1574,12 @@ index 0000000000000000000000000000000000000000..51cf0014c4229fc8671804d885b63819
 +            public boolean tick = true;
 +        }
 +
++        public Markers markers;
++
++        public class Markers extends ConfigurationPart {
++            public boolean tick = false;
```

Comment:
This should default to true to keep vanilla behavior. Having them not tick by default breaks them working as passengers.

Reply:
I made this false to keep Paper's behavior consistent with how it has been since the patch was originally applied (Jan 2022ish?). Doesn't make much difference to me, though - if it's better to have the vanilla behavior by default then I'll change it rq

Reply:
Yeah, I think we should stick to the vanilla default here. It should've been that from the beginning.